server/db/seeds.js

```diff
@@ -0,0 +1,39 @@
+use shelter;
+db.dropDatabase();
+
+const cats = [
+    {
+    name: 'Tom',
+    breed: 'Persian',
+    age: 8,
+    about: 'Mouse botherer'
+    },
+
+    {
+    name: 'Kirby',
+    breed: 'Manx',
+    age: 6,
+    about: 'Awww his little face'
+    }
+];
+
+const dogs = [
+    {
+    name : 'Sparky',
+    breed: 'Crested Chinese',
+    age: 4,
+    about: 'He zoomin'
+    },
+
+    {
+    name: 'Cara',
+    breed: 'Afghan Hound',
+    age: 3,
+    about: 'Model bitch'
+    }
+];
+
+db.animals.insertMany(cats);
```

Comment:
I had a look and it seems that we need to differentiate here. so db.cats.insertMany(cats) and the same for the dog section. this will allow us to make a call to api/cats and api/dogs

Reply:
Noted and corrected!